services/api/service.go

```diff
@@ -985,27 +1154,21 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
-	if builderIsBlacklisted {
-		log.Info("builder is blacklisted")
-		time.Sleep(200 * time.Millisecond)
-		w.WriteHeader(http.StatusOK)
-		return
-	}
-
 	// In case only high-prio requests are accepted, fail others
-	if api.ffDisableLowPrioBuilders && !builderIsHighPrio {
+	if api.ffDisableLowPrioBuilders && builderStatus != common.HighPrio {
 		log.Info("rejecting low-prio builder (ff-disable-low-prio-builders)")
 		time.Sleep(200 * time.Millisecond)
 		w.WriteHeader(http.StatusOK)
 		return
 	}
 
 	log = log.WithFields(logrus.Fields{
-		"builderHighPrio": builderIsHighPrio,
-		"proposerPubkey":  payload.Message.ProposerPubkey.String(),
-		"parentHash":      payload.Message.ParentHash.String(),
-		"value":           payload.Message.Value.String(),
-		"tx":              len(payload.ExecutionPayload.Transactions),
+		"builderStatus":    builderStatus,
+		"builderStatusStr": builderStatus.String(),
```

Comment:
May be cleaner to remove `"builderStatus":    builderStatus,` and replace `"builderStatusStr": builderStatus.String(),` by `"builderStatus": builderStatus.String(),`. Same comment [here](https://github.com/michaelneuder/mev-boost-relay/pull/1/files#diff-b5ab69e60a6fe5f5d399dbbb8c1feb340b7f43ec285fbfbb19d27ee96344cf6bL950).

Reply:
sure! i guess the value of the code is not that useful for debug logging!